web/src/App.jsx

```
@@ -1862,26 +1862,30 @@ const Sidebar = memo(({
-          <img src={beaconLogo} alt="Beacon" />
-          <span>Beacon</span>
-        </button>
+        {remoteSettings.pulseLinkEnabled && (
+          <button
+            className={`nav-item ${activePanel === 'pulselink' ? 'active' : ''}`}
+            onClick={() => setActivePanel('pulselink')}
+            title="PulseLink"
+            aria-label="PulseLink"
+            aria-current={activePanel === 'pulselink' ? 'page' : undefined}
+          >
+            <img src={logo} alt="PulseLink" />
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)